EESSI-pilot-install-software.sh

```diff
@@ -362,6 +362,12 @@ fail_msg="Installation of Nextflow failed, that's unexpected..."
 $EB -r --from-pr 16531 Nextflow-22.10.1.eb
 check_exit_code $? "${ok_msg}" "${fail_msg}"
 
+echo ">> Installing requirements for the Nextflow mag 20221110 pipeline..."
+ok_msg="The full Nextflow pipeline is installed, the work must flow..."
+fail_msg="Installation of the Nextflow pipeline failed, that's unexpected..."
+$EB --robot --from-pr 16576,16577,16589,16590,16591,16592,16612,16613,16615,16617
```

Comment:
@smoretti This should be trimmed to:

```
$EB --from-pr 16613 --robot
```

As long as all PRs required by https://github.com/easybuilders/easybuild-easyconfigs/pull/16613 are merged (most, if not all, already are), then that's sufficient.

Reply:
Done